metasploit-framework.gemspec

```diff
@@ -65,7 +65,7 @@ Gem::Specification.new do |spec|
   # Metasploit::Credential database models
   spec.add_runtime_dependency 'metasploit-credential'
   # Database models shared between framework and Pro.
-  spec.add_runtime_dependency 'metasploit_data_models'
+  spec.add_runtime_dependency 'metasploit_data_models', '3.0.10'
```

Comment:
If we want to keep releasing new gems with this code we should probably do that under a new major version number (`4.x.x`). That way, we can use a version restriction here for `<4.0`) in case we need to pull in a bug fix for metasploit data models (and to make life for Pro easier).

Reply:
Yep, next revision I think we should bump so we can segregate this from the 4.x-compatible stream.

Reply:
Pro is currently on `metasploit_data_models 2.x` stream however once msf5 reaches Pro this recommendation would be good to support.